### PR TITLE
Add `remark-lint-double-link` plugin

### PR DESCRIPTION
**Pull request by Scrum**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ exports.plugins = [
 	[require('remark-lint-heading-style'), 'atx'],
 	[require('remark-lint-link-title-style'), '\''],
 	require('remark-lint-list-item-bullet-indent'),
+	require('remark-lint-double-link'),
```
Comment (sindresorhus) [inline config.js:21]: This is not an official plugin

**Comment by sindresorhus** [comment]
I also noticed that the matching is quite naive. It's just a string comparison. I would recommend doing some simple normalization. See https://github.com/sindresorhus/normalize-url

**Comment by Scrum** [comment]
> I also noticed that the matching is quite naive. It's just a string comparison. I would recommend doing some simple normalization. See https://github.com/sindresorhus/normalize-url

Thank you, great offer

File: config.js (hunk 1 above)
Comment (Scrum) [reply config.js]: fixed

**Comment by sindresorhus** [comment]
Thanks. I'll do a release when the other open PR is merged.

**Comment by Scrum** [comment]
> Thanks. I'll do a release when the other open PR is merged.

Publish v0.1.1

**Pull request merged**